### ISSUE TITLE
`timestamptz` typed as `any`

### ISSUE THREAD

**Issue by jgonera** [rmp135/sql-ts#74]
Despite https://github.com/rmp135/sql-ts/blob/518986e59602df6403352b6a76b4829ec7ed1024/src/TypeMap.ts#L5 `timestamptz` columns are typed as `any` for me in all my tables.

Sample table:

```sql
CREATE TABLE groups_securities (
    created_at timestamp with time zone NOT NULL DEFAULT now(),
    updated_at timestamp with time zone NOT NULL DEFAULT now(),
    group_id uuid NOT NULL REFERENCES groups(id),
    security_id uuid REFERENCES securities(id),
    portfolio_id uuid REFERENCES portfolios(id),
    CONSTRAINT groups_securities_pkey PRIMARY KEY (portfolio_id, security_id)
);
```

My config:

```ts
  {
    client: 'pg',
    connection: {
      host,
      user,
      password,
      database,
    },
    excludedTables: ['public.schema_version'],
    tableNameCasing: 'pascal',
    typeMap: {
      string: ['numeric'],
    },
  }
```

When I add `Date: ['timestamptz']` to `typeMap`, everything works (but I'm assuming it should work without that).

I use `sqlts.toTypeScript` because I generate the config programmatically.

**Comment by rmp135** [comment]
I've published a new version (1.9.0) that should include these changes.

**Comment by jgonera** [comment]
Oh, I didn't notice that the latest version wasn't latest `master` code. Thank you!
